src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LbfgsStatics.cs

```diff
@@ -0,0 +1,179 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Calibration;
+using Microsoft.ML.Runtime.Learners;
+using Microsoft.ML.Runtime.Training;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+
+namespace Microsoft.ML.Trainers
```

Comment:
>Microsoft.ML.Trainers [](start = 10, length = 21)

change namespace

Reply:
discussion on the other comment. 

---
In reply to: [221111414](https://github.com/dotnet/machinelearning/pull/1084#discussion_r221111414) [](ancestors = 221111414)